docs/refs/api/catalogd-webserver.md

```diff
@@ -0,0 +1,88 @@
+# catalogd webserver
+
+[catalogd](https://github.com/operator-framework/catalogd), the OLM v1 component for making catalog contents available on cluster, includes
+a web server that serves catalog contents to clients via a HTTP(S) endpoint.
+
+The endpoint to retrieve this information is provided in the `.status.contentURL` of a `ClusterCatalog` resource.
+As an example:
+
+```yaml
+    contentURL: https://catalogd-service.olmv1-system.svc/catalogs/operatorhubio/all.json
+```
+
+!!! note
+
+    The value of the `.status.contentURL` field in a `ClusterCatalog` is an arbitrary string value and can change at any time.
```

Comment:
```suggestion
    The value of the `.status.contentURL` field in a `ClusterCatalog` resource is an arbitrary string value and can change at any time.
```

Reply:
Addressed in https://github.com/operator-framework/operator-controller/pull/1331/commits/de1767ea0edfa05872092fc0aee3dda0616bd305